data/mods/PK_Rebalance v0.75/pk_critter_zombie.json

```diff
@@ -0,0 +1,939 @@
+    "vision_day":25,
+    "vision_night":50,
+    "hp":90,
+    "death_function":"NORMAL",
+    "special_attacks":[{
+            "type" : "bite",
+            "cooldown" : 5
+        }, ["GRAB", 4]],
+    "description":"The permanent shadow around this gaunt figure makes it hard to make out.  What you can see appears translucent and shimmering.  It is disoriented in light, and actively avoids fire.",
+    "flags":["HARDTOSHOOT", "SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "GROUP_BASH", "POISON", "BLEED", "NO_BREATHE", "REVIVES", "BONES", "PUSH_MON", "NIGHT_INVISIBILITY", "SWIMS"],
+    "categories":["CLASSIC"],
+    "anger_triggers":["PLAYER_CLOSE", "FRIEND_DIED", "STALK", "HURT"],
+    "placate_triggers":["FIRE"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_hulk_pk",
+    "name": "zombie tank",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"magenta_white",
+    "size":"HUGE",
+    "material":"flesh",
+    "diff":60,
+    "aggression":100,
+    "morale":100,
+    "speed":150,
+    "melee_skill":7,
+    "melee_dice":2,
+    "melee_dice_sides":20,
+    "melee_cut":5,
+    "dodge":4,
+    "armor_bash":20,
+    "armor_cut":20,
+    "armor_stab":10,
+    "armor_fire": 5,
+    "vision_day":50,
+    "vision_night":4,
+    "death_drops": "mon_zombie_hulk_death_drops",
+    "hp":666,
+    "death_function":["NORMAL"],
+    "special_attacks":[["SMASH", 20], ["BIO_OP_TAKEDOWN", 30]],
+    "description":"Rumored to be the conjoining of several bodies, the massive Zombie Tank is perhaps the ultimate tool of destruction.  Making up for what it lacks in skill with brutality, it is the demolisions of the zombie scourge.",
+    "flags":["SEES", "HEARS", "SMELLS", "STUMBLES", "WARM", "BASHES", "DESTROYS", "POISON", "ATTACKMON", "LEATHER", "NO_BREATHE", "REVIVES", "BONES", "FAT", "PUSH_MON"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_zombie_gasbag_pk",
+    "name": "putrid zombie",
+    "species":"ZOMBIE",
+    "default_faction":"zombie",
+    "symbol":"Z",
+    "color":"pink_magenta",
+    "size":"MEDIUM",
+    "material":"flesh",
+    "diff":10,
+    "aggression":100,
+    "morale":100,
+    "speed":100,
+    "melee_skill":7,
+    "melee_dice":2,
+    "melee_dice_sides":13,
+    "melee_cut":0,
+    "dodge":3,
+    "armor_bash":16,
+    "armor_cut":5,
+    "armor_fire":10,
+    "armor_acid":2,
+    "vision_day":40,
+    "vision_night":4,
+    "hp":110,
+    "death_function":["GAS", "BOOMER_GLOW"],
+    "special_attacks":[{
+            "type" : "bite",
+            "cooldown" : 30
+        },
+        ["GRAB_DRAG", 12], ["GRAB", 8], ["STRETCH_ATTACK", 15]],
+    "description":"A fetid body with a featureless face that emits a noxious gas of unknown make.  Its skin knots and reforms into putty-like globs as it walks and attacks, and bullets seem to pass through its morphing form.  Upon death its body rapidly deteriorates into a vicous pink sludge reminicent of boomers.",

```

Comment:
"unknown make" -> "unknown makeup", as in chemical makeup.

The description probably shouldn't mention what happens on death. It can warn of it by saying that it stinks like a boomer or occasionally barfs up bile on itself or something like that.